src/analysis/cytoscape/py4cy/cytoscape_util.py

```diff
@@ -0,0 +1,98 @@
+import argparse
+from pathlib import Path, PurePath
+import pathlib
+from typing import List, Dict, Any
+from numpy import append
+import py4cytoscape as p4c
+import time
+import sys
+import os
+from requests.exceptions import RequestException
+import subprocess
+
+SPRAS_ROOT = os.path.dirname(os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__)))))
```

Comment:
Why do we need the SPRAS root directory if this is now running inside the container?